src/utilities/codemirror/codemirror-utils.ts

```diff
@@ -120,10 +120,11 @@ export function getDefaultVariableArgs(parameters: VariableDeclaration[]): strin
         return parameter.allowable_values && parameter.allowable_values.length > 0
           ? `"${parameter.allowable_values[0]}"`
           : parameter.enum_name
-            ? `${parameter.enum_name}`
+            ? `"${parameter.enum_name}"`
             : 'UNKNOWN';
       default:
-        throw Error(`unknown argument type ${parameter.type}`);
+        console.log(`Unknown argument type ${parameter.type}`);
```

Comment:
I think the util file should not be in charge of logging. It would be more reusable if the consuming functions using this function are responsible for how they report errors.

Reply:
Got it. I removed the logging and are returning a value with an error so it stand out to the users that the variable type is incorrect.